File: 389. Find the Difference/main.cpp
class Solution {
public:
    char findTheDifference(string s, string t) {
        int c=0;
        
        for(int i=0;i<(s+t).length();i++)
            c=c^ (s+t)[i];
        
        return (char)c;
    }
};

File: 389. Find the Difference/main.py
import collections
class Solution:
    def findTheDifference(self, s, t):
        """
        :type s: str
        :type t: str
        :rtype: str
        """
        
        c=0 
        for e in s+t:
            c=c^ (ord(e)-97)
            
            
            
        return chr(c+97)
